docs/weave/weave-api-spec/04-querying.md

```diff
@@ -0,0 +1,143 @@
+---
+id: weave-query-spec
+title: Querying Weave
+sidebar_label: Querying Weave
+---
+
+In this section querying Weave for data will be explained. Before diving into the details, Weave internals such as **buckets** and **paths** must be explained.
+
+## Buckets
+
+Buckets are the structures that enable accessing and writing to **Key-Value** database in an organized and controlled manner. Buckets stores the data as well as the [indexes](#Primary-indexes), [secondary indexes](#Secondary-indexes) that enables you to access the data.
+
+### Accessing buckets
+
+[//]: # 'TODO give reference to Weave/tendermint or ABCI documentation'
+
+As mentioned in the previous sections, Weave uses tendermint as consensus engine thus queries are made to data store via `abci_queries`. Therefore when you make a query you do the call to tendermint's `ABCI` protocol. For more info about underlying refer to [tendermint/abciquery](https://tendermint.com/rpc/#abciquery).
+
+Via running the JSON-RPC/HTTP call below, **kissnet** testnet could be queried so you can see an example response.
```

Comment:
**kissnet** is no more and has been replaced by **lovenet**

Reply:
They grow up so fast right :information_desk_person: